addon/model/model-position.ts

```diff
@@ -302,6 +302,19 @@ export default class ModelPosition {
     return this.parent.childAtOffset(this.parentOffset - 1) || null;
   }
 
+  //this returns true if the position is inside a text node (not right before not right after)
+  isInsideText(): Boolean{
```

Comment:
```suggestion
  isInsideText(): boolean{
```
typescript prefers a type here